dev/com.ibm.websphere.javaee.cdi.1.2/bnd.bnd

```diff
@@ -34,18 +34,19 @@ Import-Package:  javax.el; version="[3.0.0,4.0.0)", \
  javax.enterprise.inject.spi;version="1.1", \
  javax.enterprise.util;version="1.1", \
  javax.inject;version=1, \
- javax.interceptor;version="1.2"
+ javax.interceptor;version="1.2", \
+ com.ibm.ws.cdi.liberty
 
 Include-Resource: \
+  META-INF=resources/META-INF, \
   @${repo;javax.enterprise:cdi-api;1.2}!/beans_1_0.xsd, \
   @${repo;javax.enterprise:cdi-api;1.2}!/beans_1_1.xsd
```

Comment:
I would strongly suggest not changing this for CDI 1.2 and 2.0 API bundles where it's not needed